pkg/filesystem/local_directory_unix.go

```diff
@@ -460,3 +463,53 @@ func (d *localDirectory) Apply(arg interface{}) error {
 		return syscall.EXDEV
 	}
 }
+
+func (d *localDirectory) Mount(mountpoint path.Component, source string, fstype string, options int) error {
+	if options != 0 {
```

Comment:
There is no strict requirement to make the API 'complete'. It needs to be complete enough for our use case. Be sure to just remove the `options` parameter.